reliquery/tests/test_storage.py

```diff
@@ -55,7 +55,7 @@ def test_use_s3_when_getting_storage_with_config_having_s3_type(tmpdir):
 
     storage = get_storage_by_name("s3", tmpdir)
     assert type(storage) == S3Storage
-    assert storage.signed == True
+    assert storage.signed is True
```

Comment:
this should be == , curious if the linter is saying otherwise.

Reply:
Changed them back